setup.py

```diff
@@ -36,10 +36,10 @@
 import vsc.install.shared_setup as shared_setup
 from vsc.install.shared_setup import ag, kh, jt, sdw
 
-VSC_INSTALL_REQ_VERSION = '0.9.19'
+VSC_INSTALL_REQ_VERSION = '0.10'
 
 PACKAGE = {
-    'version': '2.4.18',
+    'version': '2.5',
```

Comment:
`2.5.0`, to keep it consistent with other versions like `2.4.18` and the upcoming `2.5.1`

'digits' is not exactly right, but you get the idea